OpenRA.Mods.Common/Traits/Radar/AppearsOnRadar.cs

```diff
@@ -45,27 +41,24 @@ protected override void Created(Actor self)
 			modifier = self.TraitsImplementing<IRadarColorModifier>().FirstOrDefault();
 		}
 
-		public IEnumerable<Pair<CPos, Color>> RadarSignatureCells(Actor self)
+		public void RadarSignatureCells(Actor self, List<Pair<CPos, Color>> destinationBuffer)
```

Comment:
`ref` please on `destinationBuffer` to explicitly show that it's going to get modified.